.eslintrc.json

```diff
@@ -6,6 +6,8 @@
     },
     "extends": [
         "airbnb-typescript/base"
+        // TODO: Enable this after typescript conversion?
+        // "plugin:@typescript-eslint/recommended"
```

Comment:
If we don't like this, let me know and I'll drop this commit from the branch before merge.